src/Reflectivity/BreakpointHit.class.st

```diff
@@ -0,0 +1,18 @@
+Class {
+	#name : #BreakpointHit,
+	#superclass : #BreakpointAnnouncement,
+	#instVars : [
+		'valueOrNil'
```

Comment:
Can it be a meaningful name? What is stored inside?

Reply:
I'm guessing this variable is meant for the subclass VariableBreakpoint, to store the value the variable has at the moment its breakpoint triggers.